styles/style.css

```diff
@@ -6,3 +6,145 @@ Green: #00B172
 White: #FFF
 
 */
+
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  margin: 0;
+}
+
+header {
+  position: fixed;
+  top: 0;
+}
+
+.headImage {
+  background-image: url(../images/landing.jpg);
+  background-position: 0 0;
+  background-repeat: no-repeat;
+  background-size: 100%;
+  height: 67vw;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  text-align: center;
+}
+
+header #greenLogo {
+  margin-left: 20px;
+  width: 150px;
+}
+
+.headImage h1 {
+  color: white;
+  font-size: 50px;
+  margin-bottom: 0px;
+}
+
+.headImage > p {
+  color: white;
+  font-size: 20px;
+}
+
+.nav-bar {
+  height: 80px;
+  width: 100%;
+  background-color: white;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-direction: row;
+  position: fixed;
+}
+
+.nav-bar-links > ul {
+  width: 300px;
+  list-style: none;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-right: 20px;
+}
+
+.nav-bar-links > ul > li > a {
+  text-decoration: none;
+  color: black;
+}
+
+.center-part > h2 {
+  margin-left: 540px;
+  margin-top: 40px;
+  margin-bottom: 15px;
+  border-bottom: 4px solid #00b172;
+  width: 220px;
+}
```

Comment:
Try to centralize the h2 using other techniques, like display flex, etc...
using margins for big alignments like this is not a  good practice